cookie_source.go

```diff
@@ -0,0 +1,86 @@
+package mint
```

Comment:
For consistency with the other files, this should be `cookie-source.go` (reserving underscores for `_test`).